docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -1,21 +1,37 @@
 import * as React from 'react'
 import Scrollbars from 'react-custom-scrollbars'
-import { Chat, Divider } from '@stardust-ui/react'
-
+import { Chat, Divider, ChatMessage } from '@stardust-ui/react'
 import { ChatData, ChatItemTypes, generateChatProps } from './services'
 
+const ariaLiveStyle: React.CSSProperties = {
```

Comment:
what these styles are for?

Reply:
for an aria-live element, it is not visible but needed for a screen reader to narrate message preview